packages/shared/components/popups/WalletFinderPopup.svelte

```diff
@@ -67,12 +68,14 @@
                     return
                 }
 
-                await recoverAndLoadAccounts(currentAccountGapLimit, currentAddressGapLimit)
```

Comment:
This change is for fixing the broken logic now that `recoverAndLoadAccounts` does **NOT** exist anymore.